scripts/build.ps1

```diff
@@ -23,41 +24,41 @@ Push-Location (Join-Path $workpath "v8build\v8")
 # Generate the build system
 $gnargs = 'v8_enable_i18n_support=false is_component_build=false v8_monolithic=true v8_use_external_startup_data=false treat_warnings_as_errors=false'
 
-# commenting this out for now since Android build is at android/build.sh
-
-# if ($Configuration -like "*android") {
-#     $gnargs += ' use_goma=false target_os=\"android\" target_cpu=\"' + $Platform + '\"'
-# }
-# else {
-if (-not ($UseLibCpp)) {
-    $gnargs += ' use_custom_libcxx=false'
-}
+if ("android linux mac".contains($AppPlatform)) {
+    $gnargs += ' v8jsi_enable_napi=false use_goma=false target_os=\"' + $AppPlatform + '\"'
+} else {
+    if (-not ($UseLibCpp)) {
+        $gnargs += ' use_custom_libcxx=false'
+    }
 
-if ($AppPlatform -eq "uwp") {
-    # the default target_winuwp_family="app" (which translates to WINAPI_FAMILY=WINAPI_FAMILY_PC_APP) blows up with too many errors
-    $gnargs += ' target_os=\"winuwp\" target_winuwp_family=\"desktop\"'
+    if ($AppPlatform -eq "uwp") {
+        # the default target_winuwp_family="app" (which translates to WINAPI_FAMILY=WINAPI_FAMILY_PC_APP) blows up with too many errors
+        $gnargs += ' target_os=\"winuwp\" target_winuwp_family=\"desktop\"'
+    }
 }
 
 $gnargs += ' target_cpu=\"' + $Platform + '\"'
 
-if ($UseClang) {
-    #TODO (#2): we need to figure out how to actually build DEBUG with clang-cl (won't work today due to STL iterator issues)
-    $gnargs += ' is_clang=true'
-}
-else {
-    $gnargs += ' is_clang=false'
+if (!"android linux mac".contains($AppPlatform)) {

```

Comment:
>"android linux mac".contains($AppPlatform) [](http://example.com/codeflow?start=5&length=42)

nit: This check is repeated many times .. could have a single flag, say 'isWin' defined on top ..